docs/Documentation/openvino_ecosystem.md

```diff
@@ -12,70 +12,94 @@
    openvino_docs_tuning_utilities
    workbench_docs_Workbench_DG_Introduction
```

Comment:
can we remove workbench_docs_Workbench_DG_Introduction from toctree for master branch?

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16050/commits/343622d822df5737c086006d0a9dcbb0a2933221